pantheon_advanced_page_cache.info.yml

```diff
@@ -2,4 +2,4 @@ name: Pantheon Advanced Page Cache
 description: 'Advanced page cache capabilities for Pantheon.'
 package: Performance and scalability
 type: module
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^9 || ^10
```

Comment:
Is the new class available in Drupal 9.0? I am guessing it is not; `^9` will have to specify a version of Drupal that definitely has it. I'd say that `^9.4` is probably safe, but please confirm.

Reply:
It does exist from 9.0 as per the docs https://api.drupal.org/api/drupal/vendor%21symfony%21http-kernel%21Event%21ResponseEvent.php/class/ResponseEvent/9.0.x

Reply:
As I mentioned in the other PR, `^9.4 || ^10` is safer in case there are other Drupal 10.0.x APIs that we want to start using (or need to start using) in the 2.x branch.